.circleci/ds054_fasttrack_outputs.txt

```diff
@@ -14,6 +14,8 @@ fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-confounds_regres
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-confounds_regressors.tsv
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-preproc_bold.json
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-preproc_bold.nii.gz
+fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_from-native_to-T1w_mode-image_xfm.txt
```

Comment:
One possibility here, which I think is very unclear from the BIDS-Derivs draft of transforms, is whether the "default" value can be omitted (similarly to why we omit `space-` when writing the bold file):


```
fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_to-T1w_mode-image_xfm.txt
```

WDYT @effigies ?

Reply:
that also covers the inverse:
```
fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_from-T1w_mode-image_xfm.txt
```

Reply:
I think I'd rather be explicit. I see the sense in not adding an unnecessary `space-scanner` to a file that hasn't been moved into another space, but a transformation no longer has its own implicit space.